<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indoor Analysis System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Inputs"/>
    <addaction name="actionSave_Segments"/>
    <addaction name="actionLoad_Segments"/>
    <addaction name="actionSave_Object_Model"/>
    <addaction name="actionSave_Scenes"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuUpdate_Object_Model">
     <property name="title">
      <string>Update Object Model</string>
     </property>
     <addaction name="actionUpdate_Color_Model"/>
    </widget>
    <widget class="QMenu" name="menuUnify_Label">
     <property name="title">
      <string>Unify Label</string>
     </property>
     <addaction name="actionUse_Color_and_Size"/>
     <addaction name="actionMannually"/>
    </widget>
    <addaction name="actionRegionGrow"/>
    <addaction name="menuUnify_Label"/>
    <addaction name="menuUpdate_Object_Model"/>
    <addaction name="actionUpdate_Label"/>
    <addaction name="separator"/>
    <addaction name="actionIterate"/>
   </widget>
   <widget class="QMenu" name="menuReplay">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCustom_Color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuReplay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Inputs">
   <property name="text">
    <string>Open Inputs</string>
   </property>
  </action>
  <action name="actionSave_Segments">
   <property name="text">
    <string>Save Labels</string>
   </property>
  </action>
  <action name="actionRegionGrow">
   <property name="text">
    <string>RegionGrow</string>
   </property>
  </action>
  <action name="actionUpdate_Label">
   <property name="text">
    <string>Update Label</string>
   </property>
  </action>
  <action name="actionIterate">
   <property name="text">
    <string>Iterate</string>
   </property>
  </action>
  <action name="actionSave_Object_Model">
   <property name="text">
    <string>Save Objects</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionUpdate_Color_Model">
   <property name="text">
    <string>Update Color Model</string>
   </property>
  </action>
  <action name="actionUse_Color_and_Size">
   <property name="text">
    <string>Use Color and Size</string>
   </property>
  </action>
  <action name="actionMannually">
   <property name="text">
    <string>Mannually</string>
   </property>
  </action>
  <action name="actionSave_Scenes">
   <property name="text">
    <string>Save Scenes</string>
   </property>
  </action>
  <action name="actionCustom_Color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom Color</string>
   </property>
  </action>
  <action name="actionLoad_Segments">
   <property name="text">
    <string>Load Labels</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
